storage-proofs/core/src/merkle/tree.rs

```diff
@@ -105,8 +104,7 @@ impl<
 
         let proof = self.inner.gen_cached_proof(i, rows_to_discard)?;
 
-        // For development and debugging.
-        //assert!(proof.validate::<H::Function>().unwrap());
+        debug_assert!(proof.validate::<H::Function>().unwrap());
```

Comment:
Minor, but the `expect` is good too.  Might as well make it consistent here too.

Reply:
Woops, my bad. Will update and force push.